.github/workflows/pull-request-checks.yml

```diff
@@ -18,6 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: peternied/check-pull-request-description-checklist@v1.1
+        if: github.actor != 'dependabot[bot]'
```

Comment:
Looks like this was inside `${{` `}}` previously, have you been able to test that this works in your repo?
https://github.com/opensearch-project/OpenSearch/blob/bad49efab29bd5161458f182b87c5d78d8de3cac/.github/workflows/dependabot_pr.yml#L10

Reply:
@peternied Yes, I tested this on my fork by pushing the change to main and re-basing one of the existing [dependabot PRs](https://github.com/cwperks/OpenSearch/pull/149). Should I add in the curly braces?